RockPaperScissors/RockPaperScissors.cs

```diff
@@ -6,20 +6,59 @@ class Program
     {
         public static void Main()
         {
-            Console.WriteLine("Enter hand 1:");
+            Console.WriteLine("Enter your hand:");
             string hand1 = Console.ReadLine().ToLower();
-            Console.WriteLine("Enter hand 2:");
-            string hand2 = Console.ReadLine().ToLower();
+            string hand2 = ComputerHandGenerator();
             Console.WriteLine(CompareHands(hand1, hand2));
 
             // leave this command at the end so your program does not close automatically
             Console.ReadLine();
         }
-        
+
         public static string CompareHands(string hand1, string hand2)
         {
+            String Verdict = "";
+
+            //Fixes for scissor text
+            if (hand1.ToLower() == "scissor")
+            {
+                hand1 = "scissors";
+            }
+
             // Your code here
-            return hand1 + ' ' + hand2;
+            if (hand1 == hand2)
+            {
+                Verdict = "You chose "+hand1+". The Computer chose "+ hand2+"... It's a tie!";
+            }
+            if ((hand1.ToLower() == "rock" && hand2.ToLower() == "scissors") | (hand1.ToLower() == "scissors" && hand2.ToLower() == "paper") | (hand1.ToLower() == "paper" && hand2.ToLower() == "rock"))
```

Comment:
Good thinking with string comparing, but you already lower cased your input in the Main method.